pkg/proxy/userspace/rlimit_notwindows.go

```diff
@@ -0,0 +1,25 @@
+// +build linux darwin freebsd openbsd
```

Comment:
As a convention.  And this should be `// +build !windows` for now (assume presence of rlimit outside of windows).